docs/install/command-line.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Command-line
 description: Using IPFS through the command-line allows you to do everything that IPFS Desktop can do, but at a more granular level since you can specify which commands to run. Learn how to install it here.
+current-ipfs-version: v0.12.0
 ---
```

Comment:
Commenting here so we get a thread & trace some research:

> Needs small fixes but overall I think it'd be a quick win here if the maintainers are OK with it. Another approach would be to adopt some proper templating engine for docs generation and use ipfs tag as an input but that's a major undertaking compared to this change so I think we can put it off for now.

Agreed, thanks for the feedback @galargh,

FWIW:

There is also a [templating feature](https://vuepress.vuejs.org/guide/using-vue.html#templating) in vuepress,
but the syntax seems to break markdown URLs. This won't be parsed as a markdown link:

```md
[my link](https://mysite.com/{{ $frontmatter.myVariable }})
```

And [we can't use templating in code blocks](https://github.com/vuejs/vuepress/issues/413#issuecomment-472477187).

We'd have to write vuepress plugins, which would be more time-consuming & invasive.